tests/e2e/specs/in-game/game-over/rankedMatches.spec.js

```diff
@@ -214,6 +214,8 @@ describe('Creating And Updating Ranked Matches', () => {
       p1Hand: [],
       p1Points: [],
       p1FaceCards: [],
+      topCard: Card.SIX_OF_SPADES,
+      secondCard: Card.FOUR_OF_CLUBS,
```

Comment:
Test didn't care what these cards were, but legacy API defaults to leaving 2 cards in the deck when you don't specify them but you set `deck: []`, whereas the GameState API treats that as scrapping the entire deck. Setting them explicitly here so both API's will behave the same for this test